deployment/gunicorn/gunicorn_conf.py

```diff
@@ -7,11 +7,7 @@
 max_requests = 240
 # defaults to 30 sec, setting to 5 minutes to fight `GreenletExit`s
 graceful_timeout = 5*60
-# cryptically, setting forwarded_allow_ips (to the ip of the hqproxy0)
-# gets gunicorn to set https on redirects when appropriate. See:
-# http://docs.gunicorn.org/en/latest/configure.html#secure-scheme-headers
-# http://docs.gunicorn.org/en/latest/configure.html#forwarded-allow-ips
-forwarded_allow_ips = '10.176.162.109'
```

Comment:
Pretty sure this was a bug we saw when switching to nginx on hqproxy4, where redirects were coming back as http! (This would never have worked on india.) Anyhow, this bug is fixed differently in the nginx configurations, and this wouldn't have been working now on hqproxy4 anyway, so I'm killing it.